domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/party/Party.java

```diff
@@ -29,7 +29,8 @@
         @JsonSubTypes.Type(value = Individual.class, name = "individual"),
         @JsonSubTypes.Type(value = SoleTrader.class, name = "soleTrader"),
         @JsonSubTypes.Type(value = Company.class, name = "company"),
-        @JsonSubTypes.Type(value = Organisation.class, name = "organisation")
+        @JsonSubTypes.Type(value = Organisation.class, name = "organisation"),
+        @JsonSubTypes.Type(value = Claimant.class, name = "claimant")
```

Comment:
When other change is made type indicator should be something like `party` or `unspecified`